clap_mangen/src/render.rs

```diff
@@ -1,6 +1,26 @@
 use clap::{Arg, ArgAction};
 use roff::{bold, italic, roman, Inline, Roff};
 
+pub(crate) fn option_sort_key(arg: &Arg) -> (usize, String) {
```

Comment:
This should be at the bottom of the file, not the top, so people start with the entry points that matter and dig into details as needed